js/core/biblio.js

```diff
@@ -150,7 +150,7 @@ define(
                                 .concat(refs.informativeReferences)
                                 .concat(localAliases);
                 if (refs.length) {
-                    var url = "http://specref.jit.su/bibrefs?refs=" + refs.join(",");
+                    var url = "https://specref.jit.su/bibrefs?refs=" + refs.join(",");
```

Comment:
Shouldn't we just send in `new URL("//specref.jit.su/bibrefs?refs=", window.location).toString()`

That should take the protocol from the whatever the window.location is, avoiding mixed content issues.